crates/bevy_asset/src/io/file_asset_io.rs

```diff
@@ -60,10 +60,14 @@ impl FileAssetIo {
     /// Returns the base path of the assets directory, which is normally the executable's parent
     /// directory.
     ///
-    /// If the `CARGO_MANIFEST_DIR` environment variable is set, then its value will be used
+    /// If a `BEVY_ASSET_ROOT` environment variable is set, then its value will be used.
+    ///
+    /// Else if the `CARGO_MANIFEST_DIR` environment variable is set, then its value will be used
```

Comment:
Would be good to add/update this in the docs for [AssetServer.load](https://docs.rs/bevy/latest/bevy/asset/struct.AssetServer.html#method.load) too (3rd paragraph). I think it would be more discoverable as it's used by the end user more than get_base_path.